roles/mirror-registry/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Make directory for mirror-registry to use
+  become: true
+  ansible.builtin.file:
+     path: "{{ cifw_mirror_registry_base_dir }}/registry"
+     owner: "{{ lookup('env', 'USER') }}"
+     group: "{{ lookup('env', 'USER') }}"
```

Comment:
Using lookup here to avoid any dependence on Ansible fact gathering.

Reply:
beware, `lookup()` is running on the ansible controller - so in case of remote deploy (for instance against a hypervisor) this will fail.

Reply:
I can use `ansible_user`, or `ansible_env.USER`. I just wasn't sure if we wanted to `gather_facts` all the time and opted for this. If we intend to execute from one control node against a remote host for all QE tests, then I'll change this.

Reply:
Note that `ansible_user` may not be available, it comes from the inventory. `ansible_env.USER` would be the same as the lookup afaik. The "correct" variable would be `ansible_user_id` :)